woodwork/logical_types.py

```diff
@@ -968,7 +968,7 @@ def _coerce_boolean(series, null_invalid_values=False):
 
 def _transform_boolean(series, null_invalid_values):
     boolean_inference_list = config.get_option("boolean_inference_strings")
-    boolean_inference_list.append(config.get_option("boolean_inference_ints"))
+    boolean_inference_list.append(["1", "0"])
```

Comment:
This allows users to pass in and properly cast [0,1] to boolean

Reply:
This means that columns with `"0" and "1"` won't be naturally inferred as `Boolean`/`BooleanNullable`, but if a user wants to specify `Boolean`/`BooleanNullable`, then we will appropriately mark `"0"` as `False` and `"1"` as `True`. Not including this line causes `series = series.apply(lambda x: valid.get(x, np.nan))` to replace all stringified values of `1s and 0s` with `np.nan`. Attempting to convert an `np.nan` in `transform` via `series = series.astype("bool")` evaluates all `nans` to `True`. Is my understanding correct @bchen1116?

Reply:
Yep, this is correct!